public/_redirects

```diff
@@ -14,6 +14,7 @@ https://www.asyncapi.io/* https://www.asyncapi.com/:splat 301!
 /docs/getting-started/asyncapi-documents /docs/tutorials/getting-started/asyncapi-documents 301!
 /docs/getting-started/servers /docs/tutorials/getting-started/servers 301!
 /docs/getting-started/security /docs/tutorials/getting-started/security 301!
+/docs/tutorials/streetlights /docs/tutorials/create-asyncapi-document 301!
```

Comment:
We don't have this page ready on our website and the streetlights tutorial is still visible in Docs. Are we first waiting to get these changes merged in the website first before merging this PR?

Reply:
@akshatnema yes, @alequetzalli will merge it after merging new tutorials.

and the last PR to merge will be the one that removes `streetlights`

Reply:
exactly! :D